README.md

```diff
@@ -95,6 +95,16 @@ Each Minim type provides the following attributes:
 
 Each Minim type provides the following the methods.
 
+#### elementName
+
+When called, this returns the name of the element. When subclassing a Refract primitive, create this method on the new class to set a new element name.
```

Comment:
The [specification](https://github.com/refractproject/refract-spec/blob/master/refract-spec.md#element-object) calls this `element`. Maybe we should stick with that naming, or we should update the spec? I see minim as a reference implementation and keeping consistent naming is a good idea IMO.